src/Displaz.jl

```diff
@@ -234,28 +251,36 @@ When plotting lines, the `linebreak` keyword argument can be used to break the
 position array into multiple line segments.  Each index in the line break array
 is the initial index of a line segment.
 """
-function plot3d(plotobj::DisplazWindow, position; color=[1,1,1], markersize=[0.1], markershape=[0],
+function plot3d(plotobj::DisplazWindow, position; normal=[0,0,1], color=[1,1,1],
```

Comment:
More or less decided, yes.  But you're welcome to persuade me otherwise ;-)

The issue is that there's no meaningful default normal.  You could choose one arbitrarily, but I don't think it'd help with shading in that case.  It's perfectly fine to have a shader where one of the per-vertex inputs isn't available - in that case displaz will just set it to zero.